ContractExamples/scripts/prepare-timelock.sh

```diff
@@ -9,36 +9,35 @@
 
 set -e
 
-SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+dir=$(cd `dirname $0`; pwd -P)
+cd ${dir}/..
 
 ALICE=alice
-soroban keys address $ALICE || (echo "add the account $ALICE via soroban keys generate"; exit 1)
+stellar keys address $ALICE || (echo "add the account $ALICE via stellar keys generate"; exit 1)
 BOB=bob
-soroban keys address $BOB || (echo "add the account $BOB via soroban keys generate"; exit 1)
+stellar keys address $BOB || (echo "add the account $BOB via stellar keys generate"; exit 1)
 
-set -x
+#set -x
 
-TOKEN=$(
-    soroban contract deploy \
+./scripts/build.sh
+
+stellar contract deploy \
+    --salt `date +%s` \
```

Comment:
Do we need to provide the salt?

Reply:
Nope. That was a desperate attempt to remove deployment bugs